charts/ingress-nginx/README.md

```diff
@@ -1,6 +1,8 @@
 # ingress-nginx
 
-[ingress-nginx](https://github.com/kubernetes/ingress-nginx) Ingress controller for Kubernetes using NGINX as a reverse proxy and load balancer
+Ingress controller for Kubernetes using NGINX as a reverse proxy and load balancer
```

Comment:
In order to maintain consistency, I prefer to make changes here NGINX Ingress controller.

Reply:
Do you mean to keep the link text?

Reply:
https://github.com/kubernetes/ingress-nginx/pull/8061#discussion_r776540518

Reply:
Just to clarify, this will mean changing the `Chart.yaml` file. The `helm-docs` tools uses the `description` as a source for this line => https://github.com/kubernetes/ingress-nginx/blob/main/charts/ingress-nginx/Chart.yaml#L8

Reply:
I simply changed it back to what it was in the template.